SchoolProject/src/main/java/DisplayCommand.java

```diff
@@ -0,0 +1,20 @@
+import java.util.List;
+import Interfaces.IMenuCommand;
+import models.Display;
+import models.Student;
+import models.Subject;
+
+public class DisplayCommand implements IMenuCommand{
+    Display display;
+    public DisplayCommand(Display display){
```

Comment:
Looks like the Dependency Inversion pattern is broken in this part because the DisplayCommand depends on a low-level class (Display) or vice versa... And also if you try to create a Unit test for this, probably you wont be able to mock